tests/e2e/cucumber/environment/index.ts

```diff
@@ -78,13 +83,18 @@ Before(async function (this: World, { pickle }: ITestCaseHookParameter) {
         break
     }
   })
+  const tags = pickle.tags.map((tag) => tag.name)
   if (!config.basicAuth) {
     const user = this.usersEnvironment.getUser({ key: 'admin' })
     if (config.keycloak) {
       await setAccessTokenForKeycloakOcisUser(user)
       await setAccessTokenForKeycloakUser(user)
     } else {
       await setAccessAndRefreshToken(user)
+      if (tags.includes('@ocm')) {
+        config.federatedServer = true
```

Comment:
only get token and let `using "LOCAL" server` step handle the `config.federatedServer`?
```suggestion
```

Reply:
if so how you plan to delete federatedoCIS

Reply:
same like before hook using tag
```
if (tags.includes('@ocm')) {
```

Reply:
sorry but deleteUser function need config  **config.federatedServer = true** to be set